src/server/dflycmd.cc

```diff
@@ -392,6 +338,26 @@ void DflyCmd::Expire(CmdArgList args, ConnectionContext* cntx) {
   return rb->SendOk();
 }
 
+void DflyCmd::ReplicaOffset(CmdArgList args, ConnectionContext* cntx) {
+  RedisReplyBuilder* rb = static_cast<RedisReplyBuilder*>(cntx->reply_builder());
+  string_view sync_id_str = ArgS(args, 2);
+
+  VLOG(1) << "Got DFLY REPLICAOFFSET " << sync_id_str;
+  auto [sync_id, replica_ptr] = GetReplicaInfoOrReply(sync_id_str, rb);
+  if (!sync_id)
+    return;
+
+  string result;
+  unique_lock lk(replica_ptr->mu);
+  for (size_t flow_id = 0; flow_id < replica_ptr->flows.size(); ++flow_id) {
+    const auto& streamer = replica_ptr->flows[flow_id].streamer.get();
+    if (streamer) {
+      result = absl::StrCat(result, flow_id, streamer->GetRecordCount());
+    }
+  }
+  rb->SendBulkString(result);
```

Comment:
How are you gonna use it from python? I guess if you send an array here, you don't have to parse it from python, because the client will do it automatically

Reply:
In the next PR there will be debug command for replica to get it offset, which will return the sync id and the offset.
In the test I will send first the command to replica get the sync id and its offset , after that call master with the command dfly replicaoffset with the sync id got from replica. After that I will compare the offset returned from replica and master
What are you suggesting here I did not understand, send array of what?

Reply:
Do:
```
cntx->StartArray(num_flows)
for flow in flows:
  cntx->SendLong(flow->offset)
```
  
And the python client will automatically turn it into a list of ints

Reply:
Ok I will do that